polars/polars-io/Cargo.toml

```diff
@@ -51,7 +54,7 @@ lexical-core = { version = "0.8", optional = true }
 memchr = "2.5"
 memmap = { package = "memmap2", version = "0.5.2", optional = true }
 num.workspace = true
-object_store = { version = "0.5.2", features = ["aws"], optional = true }
+object_store = { git = "https://github.com/apache/arrow-rs", default-features = false, rev = "c74665808439cb7020fb1cfb74b376a136c73259" }
```

Comment:
Can you make this optional?

Reply:
Ugh, sorry. Made optional and switched to use the published version of the crate.